### ISSUE TITLE
Make "Double tap to boost or favourite" into separate options

### ISSUE THREAD

**Issue by colorchestra** [DagAgren/toot#480]
Hey! This might be a bit nitpicky, but since the latest update the "Double tap to boost" setting has become "Double tap to boost or favourite". This is not really what I want, would it be feasible to make "Double tap to boost" and "Double tap to favourite" into separate options? I think that would suit many users better.

Thank you for the great work that you do! :)

**Comment by DagAgren** [comment]
That seems awfully overcomplicated. It seems to make more sense this way. If you tend to accidentally click one, you probably also accidentally click the other?

**Comment by colorchestra** [comment]
Hmm, not sure. I favourite many, many things every day; having to tap twice every time seems like an unnecessary obstacle in my "flow" of doing things. Accidentally favouriting something I didn't mean to has no big impact. Boosting, however, has a bigger impact, because potentially many people will see it; I want all my posts (including boosts) to be well thought through, so broadcasting something to so many people unintentionally would be pretty bad. That's why I thought this was such a great feature and it made me switch to to Toot in the first place.

Also, about "If you tend to accidentally click one, you probably also accidentally click the other?": usually when I accidentally boost something, it's not because I accidentally tapped in that place, it's because I meant to favourite and hit the wrong button. This also happens the other way around sometimes, but it's not really a problem because there are rarely posts that I want to boost but not favourite, if that makes sense.

**Comment by MsCupcake** [comment]
I agree with Colorchestra, I also only want to double tap to boost. I'm left handed and I don't accidentally hit the favourite button because it's too far to the right, so having to double tap it is extra hassle.

**Comment by a-dows** [comment]
This would be a good improvement. The impact of favs vs boosts is major—favs are only between you and the person you fav, while boosts hit everyone who follows you. An accidental fave is fleeting. An accidental boost, even momentary, could be seen by any number of people. They’re very different actions, and should be treated independently. For what it’s worth, the podcast app Overcast puts settings like this in a “nitpicky details” subpage in settings. If you’re worried about too many options, that might be a good option.
